Sources/Build/BuildOperation.swift

```diff
@@ -202,7 +205,7 @@ public final class BuildOperation: PackageStructureDelegate, SPMBuildCore.BuildS
             progressAnimation: progressAnimation
         )
         self.buildDelegate = buildDelegate
-        buildDelegate.isVerbose = isVerbose
+        buildDelegate.isVerbose = verbosity != .concise
```

Comment:
should buildDelegate also take verbosity instead of boolean?

Reply:
you think so? maybe. I bet there are more places where `isVerbose` is in use

Reply:
this one is less critical